handler/websocket.go

```diff
@@ -94,6 +94,15 @@ func (c *wsConnection) init() bool {
 			}
 		}
 
+		if c.cfg.websocketOnInitFunc != nil {
+			isValidPayload := c.cfg.websocketOnInitFunc(c.ctx, c.initPayload)
+			if !isValidPayload {
```

Comment:
If you don't need error handling, it's a bit more idiomatic to write:
```suggestion
			if ok := c.cfg.websocketOnInitFunc(c.ctx, c.initPayload); !ok {
```